theano/gof/opt.py

```diff
@@ -1770,6 +1771,8 @@ def process_node(self, fgraph, node, lopt=None):
                 raise
         if replacements is False or replacements is None:
             return False
+        if replacements is True:
+            return True
```

Comment:
I have an idea to implement it in a cleaner way. I'll update this PR later with this. Mostly, add an cleanup tag to Equilibrium opt as we did for final_optimizers.